README.md

```diff
@@ -16,15 +16,15 @@ The script requires [hubot-auth](https://github.com/hubot-scripts/hubot-auth).
 
 The script can be configured via the following environment variables (called parameters).
 
-| Parameter                           | Description |
-|-------------------------------------|-------------|
-| `TENOR_API_KEY`                     | Сlient key for privileged API access. This is the only **mandatory** parameter. |
-| `TENOR_IMG_LIMIT`                   | Fetches up to the specified number of result, but not more than **50**. By default the value of the variable and the corresponding API parameter is **20**. |
-| `TENOR_SEARCH_TERM`                 | Helps to find GIFs associated with the specified term. |
-| `ANNOUNCER_CRON_STRING`             | Allows specifying the frequency with which the script checks for nearest birthdays. The value of this parameter must follow the [Cron Format](https://github.com/node-schedule/node-schedule#cron-style-scheduling). |
-| `BIRTHDAY_CRON_STRING`              | Allows specifying the frequency with which the script writes birthday messages to users. The value of this parameter must follow the [Cron Format](https://github.com/node-schedule/node-schedule#cron-style-scheduling). |
-| `BIRTHDAY_ANNOUNCEMENT_BEFORE_CNT`  | Sets how long before the event occurs the reminder will be triggered. |
-| `BIRTHDAY_ANNOUNCEMENT_BEFORE_MODE` | Unit of time. The possible values are (the corresponding shorthands are specified in the brackets): `years` (`y`), `quarters` (`Q`), `months` (`M`), `weeks` (`w`), `days` (`d`), `hours` (`h`), `minutes` (`m`), `seconds` (`s`), `milliseconds` (`ms`). |
+| Parameter                           | Description | Default |
+|-------------------------------------|-------------|---------|
+| `TENOR_API_KEY`                     | Сlient key for privileged API access. This is the only **mandatory** parameter. | |
+| `TENOR_IMG_LIMIT`                   | Fetches up to the specified number of result, but not more than **50**. By default the value of the variable and the corresponding API parameter is **20**. | 50 |
```

Comment:
'Description' column: ``` By default the value of the variable and the corresponding API parameter is **20** ```
'Default' column: ``` 50 ```

Reply:
It's tricky. **20** is the default value in Tenor API.

Reply:
Ok, but we have following words here: ```by default the value of the variable ... is **20**```, that doesn't match the value in 'default' column and is confusing.

Reply:
Never thought about it before=) I'll fix it.

Reply:
Fixed.